src/libLookup/Lookup.cpp

```diff
@@ -3168,7 +3167,7 @@ void Lookup::SendTxnPacketToNodes(uint32_t numShards) {
       {
         lock_guard<mutex> g(m_mediator.m_ds->m_mutexShards);
         auto it = m_mediator.m_ds->m_shards.at(i).begin();
-        //[FIX-ME]: Lookup sends to NUM_NODES_TO_SEND_LOOKUP + Leader
+        // Lookup sends to NUM_NODES_TO_SEND_LOOKUP + Leader
```

Comment:
FIX ME - was added here to fix the number of total nodes to send including leader. right now its NUM_NODES_TO_SEND_LOOKUP + Leader which needs t be fixed eventually.